Dockerfile

```diff
@@ -12,4 +12,6 @@ RUN ln -s /portus/docker/crono-supervisord.conf /etc/supervisor/conf.d/crono.con
 
 ENV COMPOSE=1
 
+ADD . /portus
```

Comment:
Right now the docker-compose setup is for development purposes: the git checkout is always mounted into the running container. Hence we can even drop this directive.